mscore/ove.h

```diff
@@ -1061,7 +1061,7 @@ class Line {
       int getRightXOffset() const;
 
 private:
-      QList<Staff*> staffs_;
+      QList<Staff*> staves_;
```

Comment:
Well, it's weird to have that underscore after letters. Try putting it before letters for all cases affected.

Reply:
Okay, sorry, maybe too much. I'll reorganize `ove.h` and `ove.cpp` later.

Reply:
It is weird. But i don't it belongs to this PR.
It will clearly not cause any merge conflict.